doc-Introduction_to_Self_Service_Portal/topics/My_Services_Tab.adoc

```diff
@@ -1,16 +1,20 @@
 [[my-services-tab]]
 
-== The My Services Tab
+== My Services Tab
 
-The *My Services* tab lists all available services with a summary of information associated with each service, including the *Service Name*, its current *Power State* represented by an icon, *Number of VMs*, *Owner*, date *Created*, and *Retires* date. Additionally, you can see the chargeback cost per service for a virtual machine's month-to-date usage. You can filter or sort services by those entities, and by *Relative Cost*. 
+The *My Services* tab lists all service requests individually with a summary of information associated with each service, including the *Service Name*, its current *Power State* represented by an icon, *Number of VMs*, *Owner*, date *Created*, and *Retires* date. Additionally, you can see the chargeback cost per service for a virtual machine's month-to-date usage. You can filter or sort services by those entities, and by *Relative Cost*. 
 
-On the same screen, you have the option to edit the service name and description, edit tags, set ownership, retire the service now or at a later date, or remove the service. You can also perform power operations on virtual machines by clicking more options _(represented by three dots)_ next to the service.
+On the same screen, you have the option to edit the service name and description, edit tags, set ownership, retire the service now or at a later date, or remove the service by click more options _(represented by three dots)_ next to the service. You also have options to perform power operations, that is start, stop, or suspend, at the service level. 
+
+You can perform most of the above tasks in bulk (except editing service details and power operations). To do so, select multiple services and then click the designated button for the required task on the upper right corner of the screen.    
 
 image:sui-services.png[]
 
-You can perform most of these tasks in bulk (except editing service details and power operations). In order to do so, select multiple services and then click the designated button for the required task on the upper right corner of the screen. 
+Click on a service to see more details, including information about associated virtual machines and snapshots under *Resources*, and associated parent catalog item under *Relationships*. You can click more options _(represented by three dots)_ to perform power operations at the resource level, or you can click *View Details* (or on the virtual machine name) to see more information about the resource, including compliance, retirement, timeline of power events, utilization, Smart State analysis, disk usage, and *Power Operations*. 
```

Comment:
Should *Power Operations* be changed simply to power operations here, since all the other terms are general and not marked up as UI labels? It looks a little out of place in the build, that's all.

Reply:
There is a dedicated Power Operations button on that page but I agree it looks out of place in the sentence. Fixed.